.github/CONTRIBUTING.md

```diff
@@ -71,22 +72,44 @@ Documentation for running and updating our existing tests, as well as the code f
+- Look to make sure the backport is done correctly, with attribution to the original WP committer. 
+- If there are merge conflicts during the backport, check if they have been resolved correctly by comparing the final changes from the full PR against the corresponding WP changeset(s). Remember to fix these **in a separate commit**. If you cannot resolve the conflicts, ask for help in the [Core slack channel](https://www.classicpress.net/join-slack/) or just push your branch as-is and we'll take care of it!
 
-If you're not sure about any of that, then it's a good idea to ask first. A good way is to create an issue for the specific change you're interested in, along with links to the relevant WordPress changesets and tickets, and any other information you have about how the change works.
+### Making a Backport PR.
+When you're ready to backport a code change, make a new branch to your ClassicPress fork:
+
+- Identify the WordPress **changeset number** that you'd like to port such as `43123`.
+- Run `bin/backport-wp-commit.sh` script in your terminal/command prompt.
+
+```
+bin/backport-wp-commit.sh "changeset-number"
+```
+ to apply the change to the code. 
+ 
+ If you're porting multiple changesets, you can use the `-c` option to this script 
+
+```
+bin/backport-wp-commit.sh -c "changeset-number"
+```
+to apply all the changesets to the same branch. 
 
-Finally, when you're ready to backport a code change, identify the WordPress **changeset number** that you'd like to port, and run the `bin/backport-wp-commit.sh` script to apply the change to the code. If you're porting multiple changesets, you can use the `-c` option to this script to apply all the changesets to the same branch.
+Then push your branch for review as a PR.
 
-Using this script for all backports saves time for you and for the maintainers, and it also uses a standardized format for commit messages which makes it possible for us to track which WordPress changes we've already included.
+Using this script for all backports saves time for you and for the maintainers. As it uses a standardized format for commit messages making it possible for us to track which WordPress changes we've already included.
 
-If there are merge conflicts for your changes, the script will let you know. Please fix the conflicts **in a separate commit** so that we can take a look at the conflicts and their fixes, or if you're not comfortable resolving merge conflicts, just push your branch as-is and we'll take care of it!
+**Note:** If there are merge conflicts for your changes, the script will let you know. Please fix the conflicts **in a separate commit** so that we can take a look at the conflicts and their fixes, or if you're not comßfortable resolving merge conflicts, just push your branch as-is and we'll take care of it!
```

Comment:
Extra non-English letter inserted here:

```suggestion
**Note:** If there are merge conflicts for your changes, the script will let you know. Please fix the conflicts **in a separate commit** so that we can take a look at the conflicts and their fixes, or if you're not comfortable resolving merge conflicts, just push your branch as-is and we'll take care of it!
```